crates/forge/src/running.rs

```diff
@@ -229,35 +234,42 @@ pub(crate) fn run_test_case(
         &string_to_hint,
     );
 
-    Ok(runner.run_function(
+    match runner.run_function(
```

Comment:
We're doing `find_function` again here even though we have `func` variable available.